app/model/editions/EditionsClientCollection.scala

```diff
@@ -37,9 +37,9 @@ case class EditionsClientCollection(
 case class EditionsFrontendCollectionWrapper(id: String, collection: EditionsClientCollection)
 
 object EditionsFrontendCollectionWrapper {
-  implicit def articleFormat = Json.format[EditionsClientArticle]
-  implicit def collectionFormat = Json.format[EditionsClientCollection]
-  implicit def collectionWrapperFormat = Json.format[EditionsFrontendCollectionWrapper]
+  implicit def articleFormat: OFormat[EditionsClientArticle] = Json.format[EditionsClientArticle]
+  implicit def collectionFormat: OFormat[EditionsClientCollection] = Json.format[EditionsClientCollection]
+  implicit def collectionWrapperFormat: OFormat[EditionsFrontendCollectionWrapper] = Json.format[EditionsFrontendCollectionWrapper]
```

Comment:
I know this is what IntelliJ/scalac suggests but I think the correct type here is really `Format[Xxx]` from a user perspective.